<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeopleListViewWidget</class>
 <widget class="QWidget" name="PeopleListViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>People List</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="styleSheet">
      <string notr="true">font-weight: bold; font-size: 18px;</string>
     </property>
     <property name="text">
      <string>People</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="actorsCheckBox">
     <property name="text">
      <string>Actors</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="directorsCheckBox">
     <property name="text">
      <string>Directors</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="peopleListWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="addPersonButton">
       <property name="text">
        <string>Add Person</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="goBackButton">
       <property name="text">
        <string>Go Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
